apcd-cms/src/apps/admin_exception/templates/list_admin_exception.html

```diff
@@ -2,22 +2,49 @@
 {% load static %}
 
 {% block content %}
-
 <link rel="stylesheet" href="{% static 'apcd-cms/css/table.css' %}">
 <link rel="stylesheet" href="{% static 'apcd-cms/css/modal.css' %}">
-<link rel="stylesheet" href="{% static 'submissions/css/table.css' %}">
+<!-- Must come before submissions modal stylesheet so key names in modal are not cut off -->
+<link rel="stylesheet" href="{% static 'admin_exception/css/table.css' %}">
```

Comment:
Filter drop downs on this page are compressed right around the selected option; Wes cleaned up the filter styles really nicely, I think you can source these styles instead like is done in the admin registrations table:

`<link rel="stylesheet" href="{% static 'apcd-cms/css/table.css' %}">`

Reply:
Updated! The only issue is that the submissions table style sheet is the only thing that keeps the modal from cutting off the labels for each row. I moved both the modal and submission modal style sheet to the modal only for now until the global modal css addresses this.

Reply:
Let me know if you'd like any other changes on the modal style sheets

Reply:
I'll actually remove any applicable styles that help the modal render without cutting off text in the data list from submissions to the admin_exception stylesheet. Working on this now

Reply:
This has been updated. Thanks for the suggestion!